nginx.conf

```diff
@@ -0,0 +1,35 @@
+worker_processes auto;
+pid /run/nginx.pid;
+error_log /var/log/nginx/error.log;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+  worker_connections 768;
+}
+
+http {
+  sendfile on;
+  tcp_nopush on;
+  types_hash_max_size 2048;
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3; # Dropping SSLv3, ref: POODLE
+  ssl_prefer_server_ciphers on;
```

Comment:
when you enable options
- you need to know the impact
- why did you enable SSL protocols where do you don't have SSL cert for the sever/location below?

Reply:
Those options were there in the default config of the forked repo. The specification on my task described the actions I had to take a little vague. Including files to the final image, among them, I assumed, had to be nginx.conf. Initially I thought I had to make adjustments to it, not creating the conf file from scratch. And therefore I left those lines of code intact.

I cleaned up my current config file, to leave only necessary settings for the server.